charts/flink-operator/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+description: Intalls the Spotify-version of the Flink-on-K8S-Operator
+name: flink-operator
+version: 0.0.1
+appVersion: v0.3.0
+sources:
+  - https://github.com/spotify/flink-on-k8s-operator/releases
+maintainers:
+  - name: pjthepooh
+    email: pjhuang@nextdoor.com
+dependencies:
+  - name: flink-operator-crd
+    version: 0.3.0
```

Comment:
this needs to be 0.3.1?

Reply:
ah its a nice to have, but not critical.. and right now the tests would fail because this PR includes the 0.3.1 changes, but they haven't been merged yet so there is no downloadable chart yet anyways